src/components/layers-panel/layers-panel.spec.tsx

```diff
@@ -532,4 +532,66 @@ describe("Layers Panel", () => {
 
     expect(screen.queryByText("Warning Panel")).not.toBeInTheDocument();
   });
+
+  it("Should show 'Webscene slides cannot be loaded in Across Layers mode' warning", async () => {
+    loadMock.mockImplementation(() =>
+      Promise.resolve({
+        header: {
+          presentation: {
+            slides: [
+              {id: 'slide-1'},
+              {id: 'slide-2'}
+            ]
+          }
+        },
+        layers: [
+          {
+            id: "child-layer-id",
+            title: "child-test",
+            url: "https://child-test.url",
+          },
+        ],
+      })
+    );
+
+    callRender(renderWithTheme, {
+      layers: [{ id: "test", name: "first", url: "https://test.url" }],
+      isAddingBookmarksAllowed: false
+    });
+    // @ts-expect-error - Property 'mock' does not exist on type
```

Comment:
get rid of @ts-expect-error

Reply:
You need redeclare the component:
`const LayersControlPanelMock= LayersControlPanel as unknown as jest.Mocked<any>;`

Reply:
Done